packages/bpk-docs/package.json

```diff
@@ -68,6 +68,7 @@
     "react-native-bpk-component-button": "^2.0.0",
     "react-native-bpk-component-text": "^2.1.3",
     "react-native-bpk-component-text-input": "^1.0.3",
+    "react-native-bpk-component-card": "^1.0.0",
```

Comment:
Hang on, this isn't released yet is it? If not it should be 0.0.1 (going on 1.0.0 when it's released)

Reply:
well no, because in the package.json in card component the version is 1.0.0, if I change it to 0.0.1 lerna will complain and a lot

Reply:
Change the version in `react-native-bpk-component-card/package.json` back to `0.0.1` in this case - otherwise the first release will be `1.0.1` :)

Reply:
Thanks, will do